src/libs/actions/Session/index.ts

```diff
@@ -418,6 +428,24 @@ function beginSignIn(email: string) {
     API.read(READ_COMMANDS.BEGIN_SIGNIN, params, {optimisticData, successData, failureData});
 }
 
+/**
+ * Create Onyx update to clean up anonymous user data
+ */
+function buildOnyxDataToCleanUpAnonymousUser() {
```

Comment:
Do we really need this if we have the anonymous session (and therefore the `accountID`) just before logging in?

Reply:
Ah I see your point! Just thinking though, don't we clear all Onyx data on logging out, so all those hypothetical extra anonymous records would get cleared on next log-out anyway?